Source/Icebreaker/Services/MatchingService.cs

```diff
@@ -80,6 +80,7 @@ public async Task<int> MakePairsAndNotifyAsync()
 
                 // Fetch all db users opt-in status/lookup
                 var dbMembersLookup = await this.dataProvider.GetAllUsersOptInStatusAsync();
```

Comment:
Can we merge these 2 DB calls? Something like GetAllUsers?

Reply:
Are you suggesting that `GetAllUsers` will return `Dictionary<string, Tuple<string, string>>`? Based on how some of the other helper functions are currently written I would have to break it apart into two dictionaries at the end.